stdlib/2and3/codecs.pyi

```diff
@@ -31,7 +31,8 @@ class _IncrementalDecoder(Protocol):
 def encode(obj: _Decoded, encoding: str = ..., errors: str = ...) -> _Encoded: ...
 def decode(obj: _Encoded, encoding: str = ..., errors: str = ...) -> _Decoded: ...
 def lookup(encoding: str) -> CodecInfo: ...
-def utf_16_be_encode(obj: _Decoded, errors: str = ...) -> _Encoded: ...  # undocumented
+def utf_16_be_decode(__obj: _Encoded, __errors: str = ..., __final: bool = ...) -> _Decoded: ...  # undocumented
```

Comment:
Both of these actually return a tuple of either decoded or encoded and an int:

```
In [117]: codecs.utf_16_be_encode('x', 'strict')                                                                                                                                                       
Out[117]: (b'\x00x', 1)

In [118]: codecs.utf_16_be_decode(b'x', 'strict')                                                                                                                                                      
Out[118]: ('', 0)
```

Reply:
Oops - fixed, thanks.